spec/api/people/show_spec.rb

```diff
@@ -0,0 +1,29 @@
+# frozen_string_literal: true
+
+#  Copyright (c) 2023, Schweizer Alpen-Club. This file is part of
+#  hitobito and licensed under the Affero General Public License version 3
+#  or later. See the COPYING file at the top-level directory or at
+#  https://github.com/hitobito/hitobito
+
+require 'rails_helper'
+
+RSpec.describe "people#show", type: :request do
+  it_behaves_like 'jsonapi authorized requests' do
+    let(:params) { {} }
+    let!(:person) { people(:top_leader) }
+
+    subject(:make_request) do
+      jsonapi_get "/api//people/#{person.id}", params: params
```

Comment:
Ist das Absicht dass in ein paar der Specs zwei Slashes nach /api stehen?

Reply:
nö :upside_down_face: